### PR TITLE
Fix forgot password error display

### PR DESCRIPTION
**Pull request by pkong-ds**
ref DEV-2058

Maybe should enquire fung on `what error to show under input field` vs `what error to show in alert message`

### CODE REVIEW

File: resources/authgear/templates/en/web/authflowv2/forgot_password.html [hunk 1]
```diff
@@ -37,12 +37,6 @@ <h1 class="screen-title">{{ template "v2.page.forgot-password.default.title" }}<
             {{ template "v2.page.forgot-password.default.email-input-description" }}
           {{ end }}
         </p>
-        {{ template "authflowv2/__alert_message.html"
-          (dict
-            "Type" "error"
-            "Classname" "mt-4"
-          )
-        }}
```
Comment (louischan-oursky) [inline resources/authgear/templates/en/web/authflowv2/forgot_password.html:45]: Again, every page should have __alert_message.html, why do you keep wanting to remove it?

**Review by louischan-oursky — CHANGES_REQUESTED** [review]

File: resources/authgear/templates/en/web/authflowv2/forgot_password.html (hunk 1 above)
Comment (pkong-ds) [reply resources/authgear/templates/en/web/authflowv2/forgot_password.html:45]: Oh, the `__alert_message.html` in this file does not do anything right now. Note `Message` param is missing.

As mentioned in DEV-2058

> Currently, all errors is put under input field.

So this commit is just removing unused code

Comment (pkong-ds) [reply resources/authgear/templates/en/web/authflowv2/forgot_password.html:45]: I think a proper fix would be to distinguish what error to show in input/ what error to show in alert message. But might need fung input on this?

Comment (louischan-oursky) [reply resources/authgear/templates/en/web/authflowv2/forgot_password.html:45]: I see.

**Review by louischan-oursky — APPROVED** [review]

**Pull request merged**